tracer/test/snapshots/Iast.PathTraversal.AspNetCore2.TelemetryEnabled.verified.txt

```diff
@@ -51,15 +51,16 @@
       _dd.agent_psr: 1.0,
       _dd.iast.telemetry.executed.propagation: 1.0,
       _dd.iast.telemetry.executed.sink.path_traversal: 1.0,
-      _dd.iast.telemetry.executed.source.http.cookie.name: 1.0,
-      _dd.iast.telemetry.executed.source.http.cookie.value: 1.0,
-      _dd.iast.telemetry.executed.source.http.request.header: 1.0,
-      _dd.iast.telemetry.executed.source.http.request.header.name: 1.0,
-      _dd.iast.telemetry.executed.source.http.request.parameter: 1.0,
-      _dd.iast.telemetry.executed.source.http.request.parameter.name: 1.0,
-      _dd.iast.telemetry.executed.source.http.request.path: 1.0,
-      _dd.iast.telemetry.executed.source.http.request.path.parameter: 1.0,
-      _dd.iast.telemetry.executed.source.http.request.query: 1.0,
+      _dd.iast.telemetry.executed.source.http_cookie_name: 1.0,
+      _dd.iast.telemetry.executed.source.http_cookie_value: 1.0,
+      _dd.iast.telemetry.executed.source.http_request_header: 1.0,
+      _dd.iast.telemetry.executed.source.http_request_header_name: 1.0,
+      _dd.iast.telemetry.executed.source.http_request_parameter: 1.0,
+      _dd.iast.telemetry.executed.source.http_request_parameter_name: 1.0,
+      _dd.iast.telemetry.executed.source.http_request_path: 1.0,
+      _dd.iast.telemetry.executed.source.http_request_path_parameter: 1.0,
+      _dd.iast.telemetry.executed.source.http_request_query: 1.0,
+      _dd.iast.telemetry.request.tainted:,
```

Comment:
The amount of tainted values at the end of a request will change easily and any controller code change or even test change could lead to different values, so we just check that a tag with some value is included.